packages/react-app/src/components/Account.jsx

```diff
@@ -91,13 +91,13 @@ export default function Account({
       <Space>
       {isOwner && <Badge count={"admin"} />}
       {!isOwner && <Badge count={"user"} style={{ backgroundColor: '#52c41a' }} />}
-      {address ? (
+      {address && address != "0x0000000000000000000000000000000000000000" ? (
```

Comment:
What do you think about having a zero address check? Instead of having to type out the entire address for every check

Reply:
how can we do that, I am down.

Reply:
updated